logger/request_context.go

```diff
@@ -0,0 +1,60 @@
+package logger
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/Sirupsen/logrus"
+)
+
+type contextKey int
+
+const (
+	ContextKeyRequestContext contextKey = iota
+)
+
+type RequestContext struct {
```

Comment:
Don't think I'd make this public either. Can always change it later if it turns out to be useful, harder to make things private